### ISSUE TITLE
statsmodels.genmod.families.links.Log documentation typo

### ISSUE THREAD

**Issue by kousu** [statsmodels/statsmodels#2306]
```
 |  inverse_deriv(self, z)
 |      Derivative of the inverse of the log transform link function
 |      
 |      Parameters
 |      ----------
 |      z : array
 |          The inverse of the link function at `p`
 |      
 |      Returns
 |      -------
 |      The value of the derivative of the inverse of the logit function
 |  
```

That should be "the exponential function", right?

Version: '0.6.1'

**Comment by josef-pkt** [comment]
Yes, looks like copy-paste error  `inverse of the log function,  the exponential function`

**Comment by kiyoto** [comment]
This should be ok to close.

**Comment by josef-pkt** [comment]
Thanks for the reminde

fixed in #2496
